cryptoki/src/error/mod.rs

```diff
@@ -45,12 +45,12 @@ pub enum Error {
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self {
-            Error::LibraryLoading(e) => write!(f, "libloading error ({})", e),
-            Error::Pkcs11(e) => write!(f, "PKCS11 error: {}", e),
+            Error::LibraryLoading(e) => write!(f, "libloading error ({e})"),
```

Comment:
Is this supported by the MSRV?

Reply:
I compiled it for `1.58.1` Parsec MSRV and found no issues.